auditbeat/docs/auditbeat-modules-config.asciidoc

```diff
@@ -23,11 +23,11 @@ auditbeat.modules:
 
 - module: file_integrity
   paths:
-  - /bin
-  - /usr/bin
-  - /sbin
-  - /usr/sbin
-  - /etc
+    - /bin
+    - /usr/bin
+    - /sbin
+    - /usr/sbin
+    - /etc
```

Comment:
Hey @ftatard, thanks for opening this PR!

But current format looks valid to me. In principle for yaml lists it is enough to indentate each item at the same level as the list name. Why do you think this needs to be changed? Did it cause any problem with your config?